### ISSUE TITLE
Set wp-cli environment variable WP_CLI_PACKAGES_DIR on Windows?

### ISSUE THREAD

**Issue by monolux** [wp-cli/wp-cli#4145]
I installed wp-cli on Windows following the [Install via Composer](https://make.wordpress.org/cli/handbook/installing/#installing-via-composer) – Global require method with following command `composer global require wp-cli/wp-cli`.

It worked fine and when I type `wp --info` I get:

```
PHP binary:     D:\XAMPP\php\php.exe
PHP version:    5.6.8
php.ini used:   D:\XAMPP\php\php.ini
WP-CLI root dir:        phar://wp-cli.phar
WP-CLI packages dir:
WP-CLI global config:
WP-CLI project config:
WP-CLI version: 1.1.0
```

Now I would like to install [a wp-cli package](https://github.com/iandunn/wp-cli-rename-db-prefix) but cannot since the WP-CLI packages dir is not set or missing. When I try to install the package I get following error: `Error: Composer directory for packages couldn't be created.
`
I found out that I [can set the wp-cli environment variables](https://make.wordpress.org/cli/handbook/config/#environment-variables) however it does not say how to do this on Windows.

Is there a way to set `WP-CLI packages dir` on Windows?

**Comment by danielbachhuber** [comment]
@monolux Didn't you just open #4144 and https://wordpress.org/support/topic/set-wp-cli-environment-variable-wp_cli_packages_dir-on-windows/ ?

**Comment by monolux** [comment]
@danielbachhuber Yes, in that issue I ask how to get the WP-CLI packages dir set up automatically using the installation guide via Composer. If however that is not possible I would like to know if I could somehow set the WP-CLI packages dir **manually**, hence this second issue. If you think these two issues are the same feel free to close. I fear that if setting the WP-ClI packages dir is not possible, though I don't know, hence these two issues, perhaps I am being too careful.

**Comment by monolux** [comment]
Due to help from @schlessera and @danielhuesken on the #cli WordPress Slack channel I found out what I had to do.

Instead of setting a PATH value in the Windows environment variables I had to make a **new** variable called `WP_CLI_PACKAGES_DIR` and set the value of that variable to `C:\Users\YOUR_WINDOWS_YOURSERNAME_HERE\.wp-cli\packages`.

I also had to create the folder `packages` under `C:\Users\YOUR_WINDOWS_YOURSERNAME_HERE\.wp-cli\`.  The folder was not created automatically.

After that though Composer was complaining that PHP does not have enough memory and so I permanently increased the memory from `128M` to `512M` in the php.ini file.

Now I could install the wp-cli package just fine, ran it and it also worked perfectly.

Reference links:
https://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10

https://make.wordpress.org/cli/handbook/common-issues/#php-fatal-error-allowed-memory-size-of-999999-bytes-exhausted-tried-to-allocate-99-bytes
